services/director-v2/src/simcore_service_director_v2/cli/__init__.py

```diff
@@ -26,7 +26,11 @@
 DEFAULT_OUTPUTS_PUSH_ATTEMPTS: Final[int] = 3
 DEFAULT_TASK_UPDATE_INTERVAL_S: Final[int] = 1
 
-main = typer.Typer(name=PROJECT_NAME)
+main = typer.Typer(
+    name=PROJECT_NAME,
+    pretty_exceptions_enable=False,
```

Comment:
Cool! I recall @dyollb speaking about these. I think we should extend these to all services in the stack